xline/src/server/xline_server.rs

```diff
@@ -66,6 +70,8 @@ pub struct XlineServer {
     server_timeout: ServerTimeout,
     /// Shutdown trigger
     shutdown_trigger: Arc<Event>,
+    /// Curp shutdown trigger
```

Comment:
Is it necessary to keep `shutdown_trigger` and `curp_shutdown_trigger` in `XlineServer`? Can we remove the `shutdown_trigger`?

Reply:
it is a temporary solution, this PR contains only the shutdown logic of curp

Reply:
So we will finally remove the `shutdown_trigger` but not in this PR? If so, please add an issue to track it.

Reply:
#436